src/hotspot/share/cds/serializeClosure.hpp

```diff
@@ -48,8 +48,8 @@ class SerializeClosure : public StackObj {
   // Read/write the bool pointed to by p.
   virtual void do_bool(bool* p) = 0;
 
-  // Read/write the region specified.
-  void do_region(u_char* start, size_t size) {
+  // Iterate on the pointers from p[0] through p[num_pointers-1]
+  void do_ptrs(u_char* start, size_t size) {
```

Comment:
I think it will be more consistent if we use the same `void** p` parameter as in `do_ptr()`. The `u_char*` here is a historical oddity and should be fixed.